apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -775,9 +880,13 @@ def test(element_html, data):
     # TODO: The only wrong answer being tested is the correct answer with the first
     # block mising. We should instead do a random selection of correct and incorrect blocks.
     elif data["test_type"] == "incorrect":
-        answer = filter_multiple_from_array(
-            data["correct_answers"][answer_name], ["inner_html", "indent", "uuid"]
-        )
+        answer = deepcopy(data["correct_answers"][answer_name])
+        distractor_feedback = {
+            item["inner_html"]: item["distractor_feedback"]
```

Comment:
Are these answers being uniquely identified by their inner HTML? Would be better to set a uuid as a unique identifier (or something similar).

Reply:
Ya, I agree with you its not cool. These actually have UUIDs on them now, but this code is a relic of the time years ago when they didn't. 

Changing this to use UUIDs now would break some questions from a few years ago. Maybe its ok if we break some questions from >2 years ago to improve code quality? What do you think @nwalters512 ?

Reply:
What I mean is, there are some questions from courses run >2 years ago, that, if someone reopened them and tried to submit something else, this grading code wouldn't work.

Reply:
We already recently changed the grading of the `unordered` grading mode to use the UUIDs anyway

Reply:
Sorry, I just realized that this comment was on the `test` function, not on `grade`. 

I can fix this here, and I have done so. 

I'm going to leave it the way it is in `grade` for now unless @nwalters512 thinks we should change it even with the potential to break old (>2 year old) questions.